.githooks/post-checkout

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+YELLOW='\033[0;33m'
+RESET_COLOR='\033[0m'
+
+SRC_PRESENT=$(git diff @{-1} HEAD --exit-code --name-only -- *.scss)
```

Comment:
can we change this to `IS_SCSS_UPDATED` or something similar

Reply:
i've used more descriptive names